test/models/queued_email_test.rb

```diff
@@ -35,9 +35,9 @@ def test_verify_account_with_user
     # User.current should always be nil when the VerifyAccount email is
     # created, so this should never happen, but somehow *has* happened multiple
     # times in the wild.  It's worth testing.
-    User.current = @dick
-    QueuedEmail::VerifyAccount.create_email(@dick)
+    User.current = dick
+    QueuedEmail::VerifyAccount.create_email(dick)
     email = QueuedEmail.last
-    assert_nil(email.user)
+    assert_nil(email&.user)
```

Comment:
@nimmolo, @pellaea 
I used safe navigation to avoid the following error:
No email is created when this test is run on its own or when testing only models.  
Under those conditions:
- `QueuedEmail.last` is `nil`, and
- `assert_nil(email.user)` throws
```ruby
Minitest::UnexpectedError:         NoMethodError: undefined method `user' for nil
            test/models/queued_email_test.rb:41:in `test_verify_account_with_user'
```